mkdocs.yml

```diff
@@ -50,6 +50,7 @@ nav:
       - ISIS FAQ:                          getting-started/using-isis-first-steps/isis-faq.md
     - Archive:
       - Demos and Workshops:               getting-started/archive/isis-demos-and-workshops.md
+      - ISIS History:                      getting-started/archive/isis-history.md
```

Comment:
I insisted putting the old demos in archive because it didn't seem useful by itself. For this, I think we can leave it out of there since it still has interesting information. Maybe in Explanations->History?